### ISSUE TITLE
[10.8.0][Issue]: Failed to determine primary image aspect ratio

### ISSUE THREAD

**Issue by yoroshikun** [jellyfin/jellyfin#7617]
### Please describe your bug

This is a re-occurring error log that that has started to appear in the 10.8.0-beta.1 build.
Unsure if directly related to M1 build however have seen similar issue pop up lately that prompted me to push this up.
Still present in 10.8.0-beta.2,
Appears whenever posters are being loaded by any client.

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: Macos, Monterey, 12.3.1
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
[14:52:55] [ERR] [29] Emby.Server.Implementations.Dto.DtoService: Failed to determine primary image aspect ratio for /Users/<user>/.local/share/jellyfin/metadata/library/b6/b678c109a7dfcf7b0a41f84a3e7d18eb/poster.jpg
System.NotImplementedException: The method or operation is not implemented.
   at Emby.Drawing.NullImageEncoder.GetImageSize(String path)
   at Emby.Drawing.ImageProcessor.GetImageDimensions(BaseItem item, ItemImageInfo info)
   at Emby.Server.Implementations.Dto.DtoService.GetPrimaryImageAspectRatio(BaseItem item)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by james187a** [comment]
Can you check to see if you have a Warning level log that says "Skia not available. Will fallback to {ImageEncoder}."?

**Comment by yoroshikun** [comment]
yes i see this on boot
```bash
[21:48:38] [WRN] [1] Emby.Server.Implementations.ApplicationHost: Skia not available. Will fallback to NullImageEncoder.
```

**Comment by james187a** [comment]
Okay, so for some reason the Skia library, which is responsible for determining image aspect ratio (among other things), is not being loaded for you. I'll see if I can find out why.

**Comment by crobibero** [comment]
As far as I know we don't have an m1 specific Jellyfin build, so that may be the cause of Skia failing to load

**Comment by yoroshikun** [comment]
> As far as I know we don't have an m1 specific Jellyfin build, so that may be the cause of Skia failing to load

Yea this is a build I built myself though changing the build flag for arm64, it all build correctly as expected (well i don't know what is as expected other than it finished successfully), possibly the skia library is being skipped?
Is it meant to be exported in the built folder somewhere?

**Comment by jimdogx** [comment]
I was able to fix my local Mac OS build by changing Jellyfin.Drawing.Skia.csproj to use the Macos libraries instead of the Linux libraries:

    <PackageReference Include="SkiaSharp.NativeAssets.macOS" Version="2.88.0" />

**Comment by cvium** [comment]
Should be fixed by #7792

**Comment by waynepaulward** [comment]
Is this fixed yet or being fixed i had this problem
